std/array.d

```diff
@@ -3927,3 +3927,230 @@ unittest
     assert(appS.data == "hellow");
     assert(appA.data == "hellow");
 }
+
+/++
+Params: arr = The array elements
+
+Returns: A static array constructed from `arr`. The type of elements can be
+specified implicitly (`int[2] a = [1,2].asStatic;`) or explicitly
+(`float[2] a = [1,2].asStatic!float`).
+The result is an rvalue, therefore uses like
```

Comment:
Rvalue/lvalue are compiler terms and shouldn't be used in user-facing docs.
Better something along the lines of creates a copy.
(BTW IIRC in same cases DMD is smart enough to infer a move.)

Reply:
As long as he can say the same thing without losing precision. The "users" here are programmers, not potato farmers :-)

Reply:
done